### ISSUE TITLE
[Question] Can you update project?

### ISSUE THREAD

**Issue by KROKIk** [softpak/HOSE#53]
Hi your project is amazing, but 1.11.2 is outdated can you please update it to 1.12 or   1.13.2 ?

**Comment by maciejopalinski** [comment]
Yeah that will be so awesome. If you cannot update it, please drop a link to something similar for newer versions of the game.

**Comment by MiniDigger** [comment]
> Yeah that will be so awesome. If you cannot update it, please drop a link to something similar for newer versions of the game.

https://papermc.io
